config/locales/administrate.it.yml

```diff
@@ -0,0 +1,23 @@
+---
+en:
```

Comment:
I think this line needs to be changed to `in:`. This might be what's causing the errors on CircleCI.